### ISSUE TITLE
not showing details of x670 motherboards

### ISSUE THREAD

**Issue by RabbitChowder** [LibreHardwareMonitor/LibreHardwareMonitor#843]
![1](https://user-images.githubusercontent.com/65978077/194783955-bd104176-aac7-4eda-a7ec-ab5f511be6ac.PNG)
My mobo is runnning relatively new bios. Do I need the newest bios, or is an libre update required? Thanks so much for any help

**Comment by GlitterKill** [comment]
Same here.  Nothing detected on X670E motherboard.
![Screenshot 2022-10-14 160051](https://user-images.githubusercontent.com/88055815/195943317-5b4db9ff-1c21-4b7a-b331-5673c4202550.jpg)

**Comment by Alexx999** [comment]
For the X670E Hero it's not picking up SuperIO with the following:

Chip ID: Unknown Winbond / Nuvoton / Fintek with ID 0xD802 at 0x2E/0x2F
Chip ID: Unknown Winbond / Nuvoton / Fintek with ID 0x8883 at 0x4E/0x4F
Chip ID: Unknown ITE with ID 0x8883 at 0x4E/0x4F
Chip ID: Unknown SMSC with ID 0x8883 at 0x4E/0x4F

But I'm not sure what's next

**Comment by jstar81** [comment]
Is this issue a reason why I am unable to see any temperatures in FanControl software https://github.com/Rem0o/ as it uses by default Libre Hardware Monitor plugin?

**Comment by Alexx999** [comment]
@jstar81 yes, the PR above is required for FanControl to work with X670E Hero. However, for me, it doesn't read all temps anyway (or reads them incorrectly).

**Comment by Inverness** [comment]
I'm also experiencing this myself:
![image](https://user-images.githubusercontent.com/1755290/197228866-4fa2bb45-615e-473e-80ac-cf10426430a9.png)

I hope to see that PR merged soon.

**Comment by NerdMayhem** [comment]
Same for ASUS TUF GAMING X670E-PLUS WIFI
![image](https://user-images.githubusercontent.com/116734880/198041346-8dcb6605-009e-44e7-ac8b-fd007dec9452.png)

**Comment by nirurin** [comment]
Same for Asus Strix X670e-e. 

Any news on if this is being worked on? Seems to be for all Asus boards only, as my MSI MPG X670 worked fine.

**Comment by andino66** [comment]
I'm having the same issues. I have an Asus ROG Crosshair x670E Hero.

**Comment by jstar81** [comment]
> I'm having the same issues. I have an Asus ROG Crosshair x670E Hero.

X670E Hero has been supported for a long time, are you using latest version? Please disable (uninstall) possible conflicting / overlapping hardware monitor softwares.
